HealthcareSystemFrontends/GraphicalEditor/Service/GenericHTTPService.cs

```diff
@@ -52,5 +52,18 @@ public void AddHTTPGetRequest(String requestURL, String JSONContent)
             request.AddJsonBody(JSONContent);
             IRestResponse response = client.Post(request);
         }
+
+        public List<T> HTTPGetRequestWithObjectAsParam<T>(string requestURL, object objectParam)
```

Comment:
Post method should probably be edited at some point to take object and handle the JSON conversion like this one does.

Reply:
Yes, that is right. An issue that still continues to be unhandled...